spec/models/profile_spec.rb

```diff
@@ -319,6 +319,22 @@
     end
   end
 
+  describe '#activate_after_passing_in_person' do
+    it 'activates a profile if it passes in person proofing' do
+      profile = create(
```

Comment:
suggestion: can we try to start using `user.profiles.create` when specs create profiles?

Reply:
Changed.

Reply:
Changed.